### ISSUE TITLE
mongo db doesn't start after reboot

### ISSUE THREAD

**Issue by kabelklaus** [Yooooomi/your_spotify#170]
**Describe the bug**

If I reboot my server, the service mongo doesn't start.
I use docker compose. I try it with a change in the docker compose yml. I added "restart: unless-stopped"

**Expected behavior**

I want that the mongo service starts after reboot.
Without this service, I don't have access to my account and data.

**Additional context**

version: "3"

services:
  server:
    image: yooooomi/your_spotify_server
    restart: unless-stopped
    ports:
      - "8080:8080"
    links:
      - mongo
    depends_on:
      - mongo
    environment:
      - API_ENDPOINT=http://192.168.2.232:8080 # This MUST be included as a valid URL in the spotify dashboard (see below)
      - CLIENT_ENDPOINT=http://192.168.2.232:3000
      - SPOTIFY_PUBLIC=                                
      - SPOTIFY_SECRET= 
      - CORS=http://192.168.2.232:3000,http://localhost:3001 # all if you want to allow every origin
  mongo:
    container_name: mongo
    image: mongo:4.4.8
    restart: unless-stopped
    volumes:
      - ./your_spotify_db:/data/db

  web:
    image: yooooomi/your_spotify_client
    restart: unless-stopped
    ports:
      - "3000:3000"
    environment:
      - API_ENDPOINT=http://192.168.2.232:8080

**Screenshots**

**Shared link**

If you don't mind, you can add a Shared link to this issue by clicking **Share this page** on your YourSpotify application so that I can identify the issue quicker.

**Comment by Yooooomi** [comment]
Hello, can you share the logs of your containers please?

**Comment by vordenken** [comment]
Is there a reason that "restart=unless-stopped" isn't added to mongo but to the other 2 containers in the docker-compose.yml?
My mongo container had an error tonight and after investigating why it didn't restart automatically I found that this flag is missing...

**Comment by megacorvega** [comment]
FWIW It seems as though adding `restart: unless-stopped` to the mongo container fixes this issue for me. I was able to restart my server running docker and connect from two different clients to the web UI.

Is there a Spotify token timeout or something though? The first time this happened to me, I had `unless-stopped added` to the mongo db container, but I couldn't get to the login page to attach the account. running `docker compose down` and then bringing it back up again fixed the issue.

**Comment by sodalimes** [comment]
I have the same issue, Docker shut down and Mongo didn't start up again. 

```
2024-03-08 16:06:59 {"t":{"$date":"2024-03-08T23:06:59.678+00:00"},"s":"I",  "c":"NETWORK",  "id":4915701, "ctx":"main","msg":"Initialized wire specification","attr":{"spec":{"incomingExternalClient":{"minWireVersion":0,"maxWireVersion":17},"incomingInternalClient":{"minWireVersion":0,"maxWireVersion":17},"outgoing":{"minWireVersion":6,"maxWireVersion":17},"isInternalClient":true}}}
error from daemon in stream: Error grabbing logs: invalid character '\x00' looking for beginning of value
```

**Comment by Yooooomi** [comment]
Fixed in 1.11.0 in 0d8e1b614ff062e136f3b8c03c2f2eecc91d3564
